src/adapter/src/coord/sequencer.rs

```diff
@@ -105,6 +108,19 @@ impl<S: Append + 'static> Coordinator<S> {
         depends_on: Vec<GlobalId>,
     ) {
         event!(Level::TRACE, plan = format!("{:?}", plan));
+        if session.vars().emit_tracing_notice() {
```

Comment:
It's possible that the statement has failed before it ever reaches this point, do we care about those statements or only care about statements that make it through planning?

Reply:
Oh hmm, good question. Do you think that would be helpful? Have we ever had slowness those cases that we've wanted to debug?

Reply:
I'm not too sure to be honest. Planning can be somewhat complex for certain queries but I think it ends up being fairly quick.

Reply:
Some other places you could put it are:
- `handle_execute` in `command_handler.rs`
- `plan_statement` in `mz-adapter::coord::sql.rs`.

Reply:
Great thanks, will do.

Reply:
`handle_execute` will catch everything that makes it past parsing. If you want to include parsing then it gets a little trickier because the extended protocol allows you to parse a statement once and execute it multiple times.

Reply:
I think we can exclude parsing for now, so I'll put it in `handle_execute`, thanks for the pointer!